### ISSUE TITLE
Markdown single newlines are rendered as paragraph boundaries

### ISSUE THREAD

**Issue by srackham** [nestdotland/nest.land#275]
Single newlines are rendered as paragraph boundaries and consequently Markdown documentation not formatted correctly.

For example, compare the opening paragraph this README on Github and on nest.land:

- https://nest.land/package/drake
- https://github.com/srackham/drake/

**Comment by ebebbington** [comment]
FYI GH seem quite inconsistent with this... whilst they aren't a new paragraph in README's, they are in issues

**Comment by srackham** [comment]
@ebebbington Yes, GH is inconsistent, but they [admitted that this behaviour was a mistake back in 2014](https://about.gitlab.com/blog/2014/02/21/markdown-newline-behaviour/). The point is that the behaviour does not conform to the [Markdown specification](https://daringfireball.net/projects/markdown/syntax#p).

**Comment by t8** [comment]
Hey @srackham! So sorry for the late reply to this. 

I see your point on the opening lines of the `drake` module README. This is likely something that needs to be fixed with the markdown formatter we currently used.

CC @martonlederer

**Comment by martonlederer** [comment]
Yeah sadly we can't really fix that
